build/android/bytecode/java/org/brave/bytecode/BraveClassVisitor.java

```diff
@@ -289,6 +356,23 @@ protected void visitImpl(int version,
         super.visit(version, access, name, signature, superName, interfaces);
     }
 
+    @Override
+    public void visitInnerClass​(String name,
+                      String outerName,
+                      String innerName,
+                      int access) {
+        if (shouldDeleteInnerClass(innerName)) {
+            System.out.println("delete InnerClass " + innerName + " from " + mName);
```

Comment:
I suppose we should return here, otherwise there won't be any change in bytecode.

Reply:
Added.